test/functional/autoupdate/aup-basic.bats

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env bats
+
+# Author: Karthik Prabhu Vinod
+# Email: karthik.prabhu.vinod@intel.com
+
+load "../testlib"
+
+test_setup() {
+
+	if [ -z "$TRAVIS" ]; then
+		skip "This test is intended to run only in Travis (use TRAVIS=true to run it anyway)..."
+	fi
+
+	create_test_environment "$TEST_NAME"
+
+}
+
+test_teardown() {
+
+	if [ -z "$TRAVIS" ]; then
+		return
+	fi
+
+}
+
+@test "AUT001: Basic test, Check auto-update enable" {
+
+	echo "printing SWUPD_OPTS"
+	echo "$SWUPD_OPTS"
+	echo "printing PATH_PREFIX"
+	echo "$PATH_PREFIX"
+	sudo sh -c "ls -l '$PATH_PREFIX'/etc/"
+	sudo sh -c "ls -l '$PATH_PREFIX'/usr/lib"
+
+	# perform autoupdate disable
+	run sudo sh -c "$SWUPD autoupdate --disable --path=$PATH_PREFIX"
+	#assert_status_is "$SWUPD_OK"
+	#run sudo sh -c "ls -l '$PATH_PREFIX'/etc/systemd/system/swupd-update.service"
+	#run sudo sh -c "ls -l '$PATH_PREFIX'/etc/systemd/system/swupd-update.timer"
+	#run sudo sh -c "ls -l '$PATH_PREFIX'/usr/lib/systemd/system/swupd-update.service"
+	#run sudo sh -c "ls -l '$PATH_PREFIX'/usr/lib/systemd/system/swupd-update.timer"
+	expected_output=$(cat <<-EOM
+		Warning: disabling automatic updates may take you out of compliance with your IT policy
+		Running systemctl to disable updates
+	EOM
+	)
+	#assert_is_output "$expected_output"
+	echo "Should see values"
+	sudo sh -c "ls -l '$PATH_PREFIX'/etc/"
+	sudo sh -c "ls -l '$PATH_PREFIX'/usr/lib"
+
+
+	# perform autoupdate enable
+	run sudo sh -c "$SWUPD autoupdate --enable --path=$PATH_PREFIX"
+	#sudo sh -c "ls -la '$PATH_PREFIX'/etc/systemd/"
+	sudo sh -c "ls -la '$PATH_PREFIX'/usr/lib/systemd/system/"
+	sudo sh -c "ls -la '$PATH_PREFIX'/usr/lib/systemd/system/swupd-update.service"
+	sudo sh -c "ls -la '$PATH_PREFIX'/usr/lib/systemd/system/swupd-update.timer"
```

Comment:
/usr/lib/systemd/ is not relevant here. Check the /etc.

Reply:
removed that was for test